pages/product/[name].js

```diff
@@ -0,0 +1,117 @@
+              <img
+                key={i.url}
+                loading="lazy"
+                src={relativizeURL(i.url)}
+                onClick={() => {
+                  setSelectedImage(relativizeURL(i.url))
+                }}
+                className="h-[250px] w-auto cursor-pointer hover:bg-white"
+              />
+            ))}
+          </div>
+        </div>
+        <div className="flex w-full flex-col items-start px-10 md:w-[35%]">
+          <h1 className="mt-10 text-3xl font-bold md:mt-0">{data.name}</h1>
+          <h2 dangerouslySetInnerHTML={{ __html: data.description }} className="text-md mt-5 font-light"></h2>
+          <div className="mt-10 flex w-full flex-row justify-between">
+            <div className="flex flex-row items-center space-x-1">
+              <StarIcon className="h-[20px] w-[20px] text-gray-700" />
+              <StarIcon className="h-[20px] w-[20px] text-gray-700" />
+              <StarIcon className="h-[20px] w-[20px] text-gray-700" />
+              <StarIcon className="h-[20px] w-[20px] text-gray-700" />
+              <StarIconOutline className="h-[18px] w-[18px] text-gray-700" />
+            </div>
+            <span>36 reviews</span>
+          </div>
+          <button className="mt-5 w-full bg-black px-2 py-4 uppercase text-white">Add To Cart</button>
+          <span className="mt-5 text-lg font-medium">Care</span>
+          <span className="mt-2 font-light">This is a limited edition production run. Printing starts when the drop ends.</span>
+          <div className="mt-5 h-[1px] w-full bg-gray-300"></div>
+          <span className="mt-5 text-lg font-medium">Details</span>
+          <span className="mt-2 font-light">
+            This is a limited edition production run. Printing starts when the drop ends. Reminder: Bad Boys For Life. Shipping may take 10+ days due
+            to COVID-19.
+          </span>
+          <div className="mt-5 h-[1px] w-full bg-gray-300"></div>
+        </div>
+      </div>
+      <div className="mt-10 h-[1px] w-full bg-gray-300"></div>
+      {relatedProducts.length && (
+        <div className="relative mt-10 flex w-full flex-col">
+          <h1 className="px-5 text-2xl font-bold">Related Products</h1>
+          <div className="flex flex-row items-start overflow-x-scroll">
+            {relatedProducts.map((i) => (
+              <Link key={i.images[0].url} href={`/product${i.path}`}>
+                <img loading="lazy" key={i.images[0].url} src={relativizeURL(i.images[0].url)} className="h-auto w-[250px] cursor-pointer hover:bg-white" />
+              </Link>
+            ))}
+          </div>
+        </div>
+      )}
+      <div className="mt-10 h-[1px] w-full bg-gray-300"></div>
+    </div>
+  )
+}
+
+export default Product
+
+export async function getServerSideProps({ params }) {
+  const slug = params.name
+  const resp = await fetch(`https://rishi-raj-jain-yg-layer0-nextjs-example-default.layer0-limelight.link/l0-api/products/${slug}`)
```

Comment:
Wrong link

Reply:
The link's right, it's the link proxied by the current app (the example) when deployed, this can change to the absolute URL of what it'll be deployed to.

Reply:
Can use Env Variables.

Reply:
https://github.com/layer0-docs/layer0-nextjs-example/pull/42#discussion_r888865260